fctesting/sandbox/Dockerfile

```diff
@@ -0,0 +1,44 @@
+ARG TMP_BUILD_DIR=/tmp/build
+
+FROM rust:1.32 as build
+
+ARG TMP_BUILD_DIR
+ARG FIRECRACKER_SRC_DIR="/firecracker"
+ARG FIRECRACKER_BUILD_DIR="$FIRECRACKER_SRC_DIR/build"
+ARG CARGO_REGISTRY_DIR="$FIRECRACKER_BUILD_DIR/cargo_registry"
+ARG FIRECRACKER_VERSION="0.15.0"
+
+# Install system dependecies
+#
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends \
+        musl-tools
+
+# Add musl to rust std
+RUN mkdir "$TMP_BUILD_DIR" \
+    	&& rustup install 1.32.0 \
+	&& rustup override set 1.32.0 \
+        && rustup target add x86_64-unknown-linux-musl \
+    && cd /
```

Comment:
nit: I don't believe the `cd /` is needed here, I believe each `RUN` command starts out in the dir specified by `WORKDIR` in the Dockerfile, so you shouldn't need to change back to `/` at the end of a `RUN`